superbench/benchmarks/micro_benchmarks/cublas_function/cublas_benchmark.h

```diff
@@ -444,7 +444,7 @@ int CublasFunction::check_result(int batch_count, T1 *Result_3_0, T2 *Result_cpu
     //     |<x, y>_cpu - <x,y>_gpu|/|<x, y>_cpu|/dot_length  < eps
     int error_count = 0;
     for (int i = 0; i < static_cast<int>(m * n) * batch_count; i++) {
-        double abs_err = fabs(Result_cpu[i] - Result_3_0_host[i]);
+        double abs_err = fabs(Result_cpu[i] - (T2)(Result_3_0_host[i]));
```

Comment:
Like a bug fix?

Reply:
the overload seems to be warnings before, errors after upgrade